Add Plugin: Adopt Orphan

LuLaValva opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/LuLaValva/obsidian-adopt-orphan

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
[Copyright (C) 2020-2025 by Dynalist Inc.](https://github.com/LuLaValva/obsidian-adopt-orphan/blob/eb5a524bc269ff30b34b9253c2a8babb1ddae470/LICENSE#L1)
Update the license file.

[const file = this.app.vault.getAbstractFileByPath(filePath);](https://github.com/LuLaValva/obsidian-adopt-orphan/blob/eb5a524bc269ff30b34b9253c2a8babb1ddae470/main.ts#L70-L83)
Use the MetadataCache to get all of the links.
Your current approach will also return links that are part of codeblocks, which Obsidian ignores.

[return "Orphan Links";](https://github.com/LuLaValva/obsidian-adopt-orphan/blob/eb5a524bc269ff30b34b9253c2a8babb1ddae470/main.ts#L123)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI), product / brand names follow the casing set by the respective owner.

LuLaValva commented:
Thanks for the review @joethei, much appreciated! Fixed in `v1.0.2` of the plugin:

- `MetadataCache` made everything _much_ simpler
- Updated to use sentence case in UI

You're a hero for going through all of these, seems like more than a full time job 😅

joethei commented:
You need to make all of the requested changes, the license is still unchanged.
